src/test/java/steps/NavigationStep.java

```diff
@@ -0,0 +1,33 @@
+package steps;
+
+import org.openqa.selenium.WebElement;
+import pages.MainPage;
+import pages.NewsPage;
+
+public class NavigationStep {
+
+    MainPage mainPage = new MainPage();
+    NewsPage newsPage = new NewsPage();
```

Comment:
Why in some classes you initialize variables inside constructor but in some classes you initialize them when adding?

Reply:
Initialize all variables inside the constructor.

Reply:
fixed